src/routes/siti-agri-schema.oas.yml

```
@@ -148,6 +148,64 @@ paths:
         '404':
           description: The `pointInTime` parameter was not conform to the expected pattern.
           $ref: '#/components/responses/NotFound'
+  /SitiAgriApi/cv/landUseByBusinessParcel/sheet/{sheet}/parcel/{parcel}/sbi/{sbi}/list:
+    get:
+      tags:
+        - 'siti-agri'
+        - 'land use'
+      summary: 'List Land Use for a business.'
+      description: 'Gets details for Land Use for a business specified by SBI.'
+      operationId: 'getLandUseBySheetAndParcelAndSBI'
+      parameters:
+        - name: 'sheet'
+          in: path
+          description: 'Sheet of the business to retrieve land use for.'
+          required: true
+          schema:
+            type: string
+            examples:
+              - 'SS6528'
+              - 'SS6529'
+        - name: 'parcel'
+          in: path
+          description: 'Parcel of the business to retrieve land use for.'
+          required: true
+          schema:
+            type: string
+            examples:
+              - '5662'
+              - '5663'
+        - name: 'sbi'
+          in: path
+          description: 'SBI of the business to retrieve land use for a specific parcel on a specific sheet.'
+          required: true
+          schema:
+            type: integer
+            format: int64
+            minimum: 1
+            examples:
+              - 1000000000
+              - 1111111111
+              - 2222222222
+              - 107167406
+              - 107097991
+              - 106354662
+              - 106221605
+              - 106440483
+      responses:
+        '200':
+          description: Returns all Land Use details for the specified sheet, parcel, and SBI.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LandUseByBusinessParcelResponse'
+        '403':
+          description: >
+            Getting Land Use details is forbidden, or a WAF rule has been triggered.
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          description: The sheet, parcel, or SBI was not found (i.e. 404!).
```

This `404` _is_ actually a `404` 😅 
```suggestion
          description: The sheet, parcel, or SBI was not found.
```

---

...but wait, doesn't `defaultResponse` respond `403` to not found cases!?

---

yep, we get a real 404